cmd/restic/cmd_backup.go

```diff
@@ -365,11 +365,16 @@ func runBackup(opts BackupOptions, gopts GlobalOptions, term *termstatus.Termina
 		return err
 	}
 
-	p.V("lock repository")
-	lock, err := lockRepo(repo)
-	defer unlockRepo(lock)
-	if err != nil {
-		return err
+	if !gopts.NoLock {
+		p.V("NOT locking repository as requested")
```

Comment:
The message here does not actually match what is happening in the code.  This part of the if statement looks like it actually IS locking the repository.